lesson2_28_02_24.html

```diff
@@ -84,7 +84,7 @@
                 gender
 
             </div>
-            <div class="push with master not allow"></div>
+            <div class="branch changes"></div>
```

Comment:
good work and thank you for submitting the PR, what is this line doing , wud you mind adding some comment please.